README.md

```diff
@@ -21,8 +21,10 @@ extension.
 ## Configuration
 
  1. Modify the `locales` block in the extension configuration to suit your
-    needs. The first one is the default locale and must be the same as bolts
-    own locale:
+    needs. The locales block is in the distributed config file in the extension folder:
+    /animal/translate/config/config.yml.dist. Copy this file to /app/config/extensions and
```

Comment:
The file should be automatically copied to the /app/config/extensions by bolt when installing the extension, was it not for you?